file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -0,0 +1,70 @@
+INTERFACE zif_aff_ntty_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! $values { @link zif_aff_ntty_v1.data:co_profile }
+    "! $default { @link zif_aff_ntty_v1.data:co_profile.text }
+    ty_profile TYPE c LENGTH 20,
+
+    "! $minimum 0
+    "! $maximum 2147483647
+    ty_max_length TYPE i,
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Language Dependent</p>
+      "! Language dependent
+      "! $showAlways
```

Comment:
Just a remark: I would try to avoid `$showAlways` in booleans if `false` is the default. This would shrink the JSON file without loosing information.

Reply:
I removed the `$showAllways`. Thank you for the hint.